src/main/java/ch/cern/eam/wshub/core/services/workorders/impl/ChecklistServiceImpl.java

```diff
@@ -59,13 +59,13 @@ public ChecklistServiceImpl(ApplicationData applicationData, Tools tools, InforW
 	}
 
 	public WorkOrderActivityChecklistSignatureResult[] getSignatures(InforContext context, String workOrderCode,
-																	 String activityCode) throws InforException {
+																	 String activityCode, TaskPlan taskPlan) throws InforException {
 		MP7999_GetWorkOrderActivityCheckListDefault_001_Result getResult = getSignatureWS(context, workOrderCode, activityCode);
 		WorkOrderActivityCheckListDefaultResult workOrderActivityCheckListDefaultResult = new WorkOrderActivityCheckListDefaultResult();
 		tools.getInforFieldTools().transformInforObject(workOrderActivityCheckListDefaultResult,
 				getResult.getResultData().getWorkOrderActivityCheckListDefault());
 
-		WorkOrderActivityChecklistSignatureResult[] res = filterSignatures(workOrderActivityCheckListDefaultResult);
+		WorkOrderActivityChecklistSignatureResult[] res = filterSignatures(workOrderActivityCheckListDefaultResult, taskPlan);
```

Comment:
The line is displayed with the indentation for the non whitespace character to start beneath InforContext in Intellij. Should I change it?

Reply:
Yes, thank you. It seems that tabs are being used instead of spaces in all indented lines.